invenio_queues/ext.py

```diff
@@ -74,9 +74,14 @@ def delete(self, **kwargs):
         """Delete queue for all or specific event types."""
         self._action('delete', **kwargs)
 
-    def purge(self, **kwargs):
-        """Delete queue for all or specific event types."""
-        self._action('purge', **kwargs)
+    def purge(self, force, queues=None):
+        """Purge queue for all or specific event types."""
+        for q in queues or self.queues:
+            try:
+                self.queues[q].queue.purge()
+            except Exception:
+                if not force:
+                    raise
```

Comment:
- Should `declare`/`delete` should be rewritten the same way to keep consistency? This would imply removing `_action`.

- Should `_action` be rewritten in a way it handles this by itself?